shared/src/commonMain/kotlin/co/touchlab/kampstarter/models/BaseModel.kt

```diff
@@ -9,26 +9,29 @@ import org.koin.core.KoinComponent
 import kotlin.coroutines.CoroutineContext
 
 open class BaseModel : KoinComponent {
-    internal val mainScope = MainScope(Dispatchers.Main)
-    internal val ktorScope = MainScope(Dispatchers.Main)
+    internal val scope = MainScope(Dispatchers.Main)
+    private val ktorContext: CoroutineContext by lazy { scope.childContext() }
```

Comment:
This looks unused?

Reply:
Yeah I think I forgot to strip this when I was doing my refactor